lib/structures/NewsChannel.js

```diff
@@ -26,6 +26,14 @@ class NewsChannel extends TextChannel {
         this.rateLimitPerUser = 0;
         this.update(data);
     }
+    /**
+     * Publish a message to subscribed channels
+     * @arg {String} messageID The ID of the message
+     * @returns {Promise<Message>}
+     */
+    crosspost(messageID) {
+        return this.client.crosspost.call(this.client, this.id, messageID);
```

Comment:
Also since you changed client to be crosspostMessage, the client call should be on `crosspostMessage`. Same goes for the others